.github/workflows/lint.yml

```diff
@@ -32,7 +32,7 @@ jobs:
         with:
           python-version: '3.13'
       - name: Install SQLFluff
-        run: 'pip install -r requirements.txt'
+        run: 'pip install -r sqlfluff/requirements.txt'
```

Comment:
My initial thought was "this directory name seems too narrow" but until we have another python requirement, this is truly the only thing it exists for and we can always change it another day.